samples/MvcSample.Web/HomeController.cs

```diff
@@ -113,6 +113,22 @@ public ActionResult SaveUser(User user)
         }
 
         /// <summary>
+        /// Action that shows binding of form values to <see cref="IFormCollection"/>.
+        /// </summary>
+        public ActionResult PostForm(IFormCollection form)
+        {
+            if (!ModelState.IsValid)
```

Comment:
You are right. Checking for `form != null` would make more sense here. Will update it.